packaging/install/050-Deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: strimzi-access-operator
+  labels:
+    app: strimzi-access-operator
+  namespace: strimzi-access-operator
+spec:
```

Comment:
Not absolutely required in this PR. But should we have the `/tmp` as an empty dir with limited size similarly to other operators?

Reply:
What's the purpose of the empty dir?

Reply:
Having the /tmp as a volume allows read only root filesystem. nd having a size limit on it is something some platforms enforce.

Reply:
I've added the /tmp volume and also changed the strategy to Recreate as I don't think the Java operator sdk can handle two operators running at the same time, I think they would fight with each other.